src/sparql/ordo-replace-annotation-based-mappings.ru.jinja2

```diff
@@ -0,0 +1,28 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
```

Comment:
This is the updated file which now does multiple INSERTs (and DELETEs) in a single query, rather than generating multiple query files.